<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>setup</class>
    <widget class="QDialog" name="setup">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>528</width>
                <height>536</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QLabel" name="logoLabel">
            <property name="geometry">
                <rect>
                    <x>-10</x>
                    <y>-30</y>
                    <width>541</width>
                    <height>171</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>A Charming Font Expanded</family>
                    <pointsize>36</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                </font>
            </property>
            <property name="text">
                <string>Welcome to L-Book</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
            </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_1">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>100</y>
                    <width>501</width>
                    <height>381</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLabel" name="label_2">
                        <property name="font">
                            <font>
                                <pointsize>14</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Welcome to L-Book. The school's library manager program. It seems this is your first
                                run of this program on your computer, so we need some information before we can start.
                            </string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignJustify|Qt::AlignVCenter</set>
                        </property>
                        <property name="wordWrap">
                            <bool>true</bool>
                        </property>
                        <property name="textInteractionFlags">
                            <set>Qt::NoTextInteraction</set>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="groupBox">
                        <property name="title">
                            <string>Program Usage:</string>
                        </property>
                        <widget class="QWidget" name="layoutWidget_2">
                            <property name="geometry">
                                <rect>
                                    <x>20</x>
                                    <y>50</y>
                                    <width>421</width>
                                    <height>111</height>
                                </rect>
                            </property>
                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                <item>
                                    <widget class="QRadioButton" name="LocalButton">
                                        <property name="text">
                                            <string>Manage File Localy</string>
                                        </property>
                                        <property name="checked">
                                            <bool>true</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                                        <item>
                                            <widget class="QRadioButton" name="sharedButton">
                                                <property name="text">
                                                    <string>Manage File From:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="selectFolder">
                                                <property name="text">
                                                    <string>Select Folder</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
                <rect>
                    <x>290</x>
                    <y>490</y>
                    <width>221</width>
                    <height>41</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QPushButton" name="nextButton">
                        <property name="text">
                            <string>Next</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="exitButton">
                        <property name="text">
                            <string>Exit</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
